README.md

```diff
@@ -20,7 +20,7 @@ yarn add @rocketmakers/armstrong
 Then to use a component in your project
 
 ```tsx
-import { Button } from '@rocketmakers/armstrong';
+import { Button } from "@rocketmakers/armstrong";
```

Comment:
Why? Our prettier config insists on single quotes.

Reply:
No idea - this was automated 🤷‍♂️

Will revert